M2.el

```diff
@@ -320,8 +320,8 @@ can be executed with \\[M2-send-to-program]."
 	((save-excursion
 	   (beginning-of-line)
 	   ;; example:      ../../m2/res.m2:210:45-214:6: --source code:
-	   ;;                (1                     1)   (2       2)   (3      3)   (4      4)   (5      5)   (6      6))
-	   (looking-at "^ *\\(o+[1-9][0-9]* = \\|| \\)?\\([^:\n]+\\):\\([0-9]+\\):\\([0-9]+\\)-\\([0-9]+\\):\\([0-9]+\\):"))
+	   ;;                (1                                     1)   (2       2)   (3      3)   (4      4)   (5      5)   (6      6))
+	   (looking-at "^ *\\(o+[1-9][0-9]* = \\|| \\|-- *\\|-\\* *\\)?\\([^:\n]+\\):\\([0-9]+\\):\\([0-9]+\\)-\\([0-9]+\\):\\([0-9]+\\):"))
```

Comment:
Do you know why `\\||` was used before, as opposed to `\\|`?

Reply:
I think that only the first pipe is for "or", and the second pipe is a literal pipe, maybe for a path that's enclosed in a box?

Reply:
Ah, that makes sense. Could you add an example in the comments for the future?

Reply:
Done